### PR TITLE
refactor: Enforce expanded sensitive inputs rules (no-changelog)

### PR DESCRIPTION
**Pull request by ivov**
- [`cred-class-field-type-options-password-missing`](https://github.com/ivov/eslint-plugin-n8n-nodes-base/blob/master/docs/rules/cred-class-field-type-options-password-missing.md)
- [`node-param-type-options-password-missing`](https://github.com/ivov/eslint-plugin-n8n-nodes-base/blob/master/docs/rules/node-param-type-options-password-missing.md)

### CODE REVIEW

File: packages/nodes-base/credentials/Auth0ManagementApi.credentials.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ export class Auth0ManagementApi implements ICredentialType {
 			displayName: 'Session Token',
 			name: 'sessionToken',
 			type: 'hidden',
+			// eslint-disable-next-line n8n-nodes-base/cred-class-field-type-options-password-missing
```
Comment (ivov) [inline packages/nodes-base/credentials/Auth0ManagementApi.credentials.ts:22]: Edge case to cover later on, tracked [here](https://github.com/ivov/eslint-plugin-n8n-nodes-base/issues/168#issue-1830784996).

File: packages/nodes-base/nodes/GoToWebinar/descriptions/WebinarDescription.ts [hunk 2]
```diff
@@ -406,6 +407,7 @@ export const webinarFields: INodeProperties[] = [
 			{
 				displayName: 'Is Password Protected',
 				name: 'isPasswordProtected',
+				// eslint-disable-next-line n8n-nodes-base/node-param-type-options-password-missing
```
Comment (ivov) [inline packages/nodes-base/nodes/GoToWebinar/descriptions/WebinarDescription.ts:410]: Edge case to cover later on, tracked [here](https://github.com/ivov/eslint-plugin-n8n-nodes-base/issues/168#issuecomment-1659876267).

**Review by netroy — COMMENTED** [review]
Is there a reason to not treat `Session Token` fields as sensitive?

File: packages/nodes-base/nodes/GoToWebinar/descriptions/WebinarDescription.ts (hunk 2 above)
Comment (Joffcom) [reply packages/nodes-base/nodes/GoToWebinar/descriptions/WebinarDescription.ts:410]: Ah just found another one of these further up

File: packages/nodes-base/nodes/Cisco/Webex/descriptions/MeetingDescription.ts [hunk 3]
```diff
@@ -747,6 +748,7 @@ export const meetingFields: INodeProperties[] = [
 				displayName: 'Exclude Password',
 				name: 'excludePassword',
 				type: 'boolean',
+				typeOptions: { password: true },
```
Comment (Joffcom) [inline packages/nodes-base/nodes/Cisco/Webex/descriptions/MeetingDescription.ts:751]: Another one

**Review by Joffcom — APPROVED** [review]
Good to go.

**Comment by janober** [comment]
Got released with [`n8n@1.2.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.2.1)

**Pull request merged**